cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -2732,6 +2732,53 @@ destructors_for_temps.cpp:
 #  102|     v102_7(void)                         = AliasedUse                               : ~m?
 #  102|     v102_8(void)                         = ExitFunction                             : 
 
+generic.c:
+#    7| char const* c11_generic_1()
+#    7|   Block 0
+#    7|     v7_1(void)            = EnterFunction            : 
+#    7|     mu7_2(unknown)        = AliasedDefinition        : 
+#    7|     mu7_3(unknown)        = InitializeNonLocal       : 
+#    9|     r9_1(glval<int>)      = VariableAddress[i]       : 
+#    9|     mu9_2(int)            = Uninitialized[i]         : &:r9_1
+#   11|     r11_1(glval<char *>)  = VariableAddress[#return] : 
+#   11|     r11_2(glval<char[4]>) = StringConstant[int]      : 
+#   11|     r11_3(char *)         = Convert                  : r11_2
+#   11|     r11_4(char *)         = Convert                  : r11_3
+#   11|     mu11_5(char *)        = Store[#return]           : &:r11_1, r11_4
+#    7|     r7_4(glval<char *>)   = VariableAddress[#return] : 
+#    7|     v7_5(void)            = ReturnValue              : &:r7_4, ~m?
+#    7|     v7_6(void)            = AliasedUse               : ~m?
+#    7|     v7_7(void)            = ExitFunction             : 
+
+#   14| void enk_c11_generic_2(unsigned int, int)
+#   14|   Block 0
+#   14|     v14_1(void)                = EnterFunction          : 
+#   14|     mu14_2(unknown)            = AliasedDefinition      : 
+#   14|     mu14_3(unknown)            = InitializeNonLocal     : 
+#   14|     r14_4(glval<unsigned int>) = VariableAddress[x]     : 
+#   14|     mu14_5(unsigned int)       = InitializeParameter[x] : &:r14_4
+#   14|     r14_6(glval<int>)          = VariableAddress[y]     : 
+#   14|     mu14_7(int)                = InitializeParameter[y] : &:r14_6
+#   15|     r15_1(glval<unsigned int>) = VariableAddress[r]     : 
+#   15|     mu15_2(unsigned int)       = Uninitialized[r]       : &:r15_1
+#   16|     r16_1(glval<unsigned int>) = VariableAddress[x]     : 
```

Comment:
Note that there's a `Load[x]` missing here. The value category emitted by the frontend seems to be incorrect.

Reply:
Should be fixed by the internal PR for https://github.com/github/codeql/pull/17140